contracts/RemovalUtilsTestHarness.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.13;
+
+import {RemovalUtils, UnpackedRemovalIdV0} from "./RemovalUtils.sol";
+
+/**
+ * @dev Testbed contract for testing RemovalUtils library.
+ *
+ * Not intended for deployment in productionish environments.
+ */
+
+contract RemovalTestHarness {
+  using RemovalUtils for uint256;
+
+  function createRemovalId(bytes calldata removalData)
```

Comment:
can you add natspec comments for each fn so we don't have to worry about it right before the next audit?

Reply:
Not relevant for this test harness contract I don't think fwiw